Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/LoadWearablesByParamSystem.cs

```diff
@@ -59,9 +60,12 @@ protected override async UniTask<StreamableLoadingResult<IWearable[]>> FlowInter
             for (var i = 0; i < lambdaResponse.elements.Count; i++)
             {
                 WearableDTO wearableDto = lambdaResponse.elements[i].entity;
-                intention.Results.Add(wearableCatalog.GetOrAddWearableByDTO(wearableDto));
-            }
+                IWearable wearable = wearableCatalog.GetOrAddWearableByDTO(wearableDto);
+                var wearableThumbnailComponent = new WearableThumbnailComponent(wearable);
```

Comment:
This system also misses the resolution level as it's not clear from the definition that the thumbnail must be/will be loaded. Or some other clarification is needed